packages/core/admin/ee/admin/hooks/useLicenseLimits/useLicenseLimits.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { useFetchClient, useRBAC } from '@strapi/helper-plugin';
+import { useQuery } from 'react-query';
+import { useSelector } from 'react-redux';
+
+import { selectAdminPermissions } from '../../../../admin/src/pages/App/selectors';
+
+export function useLicenseLimits() {
+  const permissions = useSelector(selectAdminPermissions);
+  const { get } = useFetchClient();
+  const {
+    isLoading: isRBACLoading,
+    allowedActions: { canRead, canCreate, canUpdate, canDelete },
+  } = useRBAC(permissions.settings.users);
```

Comment:
in the case of RW workflows, would this mean the hook would not run if you couldn't update a user?

Reply:
Seems so. I didn't invent this.

Reply:
No previously this hook was only used for admin seats.... i think in RW we'll need to tweak it so you can pass `enabled` as a prop 🤔

Reply:
Let's do some pair programming on this, when we start implementing RBAC! 👍🏼